cmd/manager/main.go

```diff
@@ -144,85 +155,143 @@ func newRootCommand() *cobra.Command {
 			hiveNSName := utils.GetHiveNamespace()
 			log.Infof("hive namespace: %s", hiveNSName)
 
-			// Allow an env var to disable leader election, useful when testing from source with make run to eliminate the wait time.
-			var disableLeaderElection bool
-			val := os.Getenv("DISABLE_LEADER_ELECTION")
-			if val != "" {
-				disableLeaderElection, err = strconv.ParseBool(val)
-				if err != nil {
-					log.WithField("value", val).Fatal("Error parsing DISABLE_LEADER_ELECTION env var")
+			// Create and start liveness and readyness probe endpoints
```

Comment:
The implementation in the above commit only serves liveness probe. We will also need readyness probe right? for which we will need to have a separate endpoint.

Reply:
I am not sure if we *need* the readiness probe. I added it in the PR including the above commit because it was available for free: We did not have it before. The only service that the hive controllers use is the metrics service. I am not versed well enough in how metrics work to know if it would be problematic or not to have a dormant hive-controllers pod getting requests via that metrics service.

Any insight on this, @dgoodwin?

Reply:
I'm quite sure prometheus won't have any major issue with it but it might cause some metrics blips where things disappear because there's no data for them. It might also trigger an alert if the endpoint is down for some period of time, I believe that's 5m today.

Reply:
I will take that as a yes for readiness probe

Reply:
Yeah, that seems like a yes to me. In a scenario where we have a dormant hive-controllers, it seems as though there is a high likelihood of metrics blips.